src/uwtools/drivers/chgres_cube.py

```diff
@@ -29,10 +30,10 @@ def namelist_file(self):
         path = self._rundir / fn
         yield asset(path, path.is_file)
         input_files = []
-        namelist = self._driver_config["namelist"]
-        if base_file := namelist.get("base_file"):
+        namelist = self._driver_config[STR.namelist]
```

Comment:
Here and in many cases that follow in the `uwtools.drivers` package, replace magic strings with references to `STR` object members. This change has been made where the strings refer to `uwtools` concepts; where strings refer to driver-specific concepts, they've been left as-is.